### PR TITLE
Expose via GMP API temporal layer info at encoder initialization and on a per frame basis.

### PR DESCRIPTION
**Pull request by aosmond**

### CODE REVIEW

**Comment by aosmond** [comment]
See https://bugzilla.mozilla.org/show_bug.cgi?id=1917583

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ LIBDIR_NAME=lib
 SHAREDLIB_DIR=$(PREFIX)/lib
 PROJECT_NAME=openh264
 MODULE_NAME=gmpopenh264
-GMP_API_BRANCH=Firefox114_2
+GMP_API_BRANCH=Firefox135
```
Comment (joakim-tjernlund) [inline Makefile:32]: I don't see Firefox135 branch at https://github.com/mozilla/gmp-api/branches/all

Comment (aosmond) [reply Makefile:32]: There is a tag which works https://github.com/mozilla/gmp-api/releases/tag/Firefox135

Comment (joakim-tjernlund) [reply Makefile:32]: > There is a tag which works https://github.com/mozilla/gmp-api/releases/tag/Firefox135

oh, I was tricked by the word BRANCH :)

**Pull request merged**